wallet/src/lib.rs

```diff
@@ -217,6 +219,7 @@ fn http_request(req: HttpRequest) -> HttpResponse {
         match assets.contents.get(asset) {
             Some((headers, value)) => {
                 let mut headers = headers.clone();
+                headers.append(&mut security_headers());
```

Comment:
Do you have any tests in place that could check that the headers are indeed delivered? This could also help to avoid regressions in the future where the headers would no longer be delivered after a code change.